ext/debian/puppet.conf

```diff
@@ -0,0 +1 @@
+../../conf/puppet.conf
```

Comment:
Git Symlinks do not work well with Windows. Like at all. Sorry.

We don't like to see them in any cross platform repository.